.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+## Issue that this pull request solves
+
+Closes: # (issue number)
+
+## Proposed changes
+
+Brief description of what is fixed or changed
+
+- Files changed
```

Comment:
A contributor might think they need to list all paths here that changed, which can be super cumbersome (I used to be on a project that required this and I can say first hand, it's annoying). I like having "Proposed changes" and "Brief description of what is fixed or changed," but maybe we remove all the bullet points"?